### ISSUE TITLE
Wrapped components cannot be referenced via parent components.

### ISSUE THREAD

**Issue by xdc0** [mongodb-js/connect-backbone-to-react#18]
Wrapped components can't be referenced directly, since the reference ends up being the `connectBackboneToReact` and not the wrapped component. Quick example:

```
class MyComponent extends Component {
  constructor() {
    super();
    this.ref = null;
  }

  render() {
    return (<WrappedComponent ref={(ref) => this.ref = ref}/>);
  }
}
```

Where `WrappedComponent` is wrapped by `connect-backbone-react`.

`this.ref` is a `connectBackboneToReact` object instead a `WrappedComponent` object.

`react-redux` fixes this by with an option `withRef`, see here: https://github.com/reduxjs/react-redux/blob/master/docs/api.md#connectadvancedselectorfactory-connectoptions

A similar pattern can allow `connect-backbone-to-react` to expose the wrapped object reference instead the wrapper itself.

**Comment by hswolff** [comment]
Great idea! Pull request for this is most welcome!

**Comment by hswolff** [comment]
This may be good to use the [forwardRef](https://reactjs.org/blog/2018/03/29/react-v-16-3.html#forwardref-api) API!

**Comment by jetpacmonkey** [comment]
This was resolved by #20
